_includes/macros/faqs.html

```diff
@@ -0,0 +1,67 @@
+{# ==========================================================================
+
+   faqs()
+
+   ==========================================================================
+
+   Description:
+
+   Builds FAQ markup when given:
+
+   faqs                      : An array of FAQs.
+
+   options (optional)        : An object used to customize
+                               the FAQ markup.
+
+   options.title             : Title to use for FAQ markup.
+
+   options.additional_classes: Extra classes added to the FAQ markup.
+
+   ========================================================================== #}
+{% macro render(faqs, options={}) -%}
+    {% set additional_classes = options.additional_classes  %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/8bbc244e9fa25f8d4d325edb9777093efac131d9